index.js

```diff
@@ -325,24 +326,63 @@ if(!helpers.isTestEnvironment()){
 }
```

Comment:
The current `if`-statement in `app.get('/logout'` can probably be replaced by a Validator check.

The current `if`-statment in `app.get('/dashboard'` can probably be replaced by a Validator check.

`app.get('/dashboard/:locationId'` could use the Validator to check for `req.session.user` and `req.cookies.user_sid`. No need for it to validate `locationId` though since we wouldn't get into this route without one.

Reply:
I only applied it to post routes as I wasn't sure - if I do that does `sessionChecker` on line 89 still need to exist or could I also just use that on `app.get('/'`?

Reply:
line *94 I was looking at a different branch locally

Reply:
Interesting. Good point. I hadn't looked at `sessionChecker` too closely. It actually seems to me like it's doing something strange. I would have expected it to redirect to `/login` if the session was missing and otherwise to go on to `next()`. If I'm understanding this correctly. If it did that, then it could also be used in `/dashboard` and `/dashboard/:locationId`... I think.

Reply:
^I'm just looking at this again now, did that seem to be a better idea than validator for those get requests? If so I can shuffle it around, I think it's kinda left over from when `/dashboard` didn't exist, it says middleware by it though so maybe it can just go away 🤔

Reply:
@trisapeace @mariocimet  I finally have the rest of things working so was wondering about this?